settings.gradle.kts

```diff
@@ -1,9 +1,9 @@
 pluginManagement {
     repositories {
         includeBuild("build-plugin")
-        gradlePluginPortal()
         google()
         mavenCentral()
+        gradlePluginPortal()
```

Comment:
Why prefer plugins from Google or Maven Central over Gradle's Plugin Portal? How is this related to the Gradle version update?

Reply:
Because most plugin are loaded from Google and Maven repositories, this speeds up dependency resolution